<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="66" hostname="LAPTOP-H8ACE30E" name="testScenarios.UrbanLadder" tests="70" failures="0" timestamp="09 Dec 2020 16:10:33 GMT" time="84.860" errors="4">
  <testcase name="driverConfig" time="21.014" classname="testScenarios.UrbanLadder">
    <error type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;LAPTOP-H8ACE30E&#039;, ip: &#039;192.168.1.49&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_261&#039;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-H8ACE30E', ip: '192.168.1.49', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
at userDefinedLibraries.DriverSetup.driverInstantiate(DriverSetup.java:56)
at testScenarios.UrbanLadder.driverConfig(UrbanLadder.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to /192.168.1.47:5555
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
... 31 more
Caused by: java.net.ConnectException: Connection timed out: connect
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 53 more
]]>
    </error>
  </testcase> <!-- driverConfig -->
  <testcase name="driverConfig" time="21.014" classname="testScenarios.UrbanLadder">
    <error type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;LAPTOP-H8ACE30E&#039;, ip: &#039;192.168.1.49&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_261&#039;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-H8ACE30E', ip: '192.168.1.49', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
at userDefinedLibraries.DriverSetup.driverInstantiate(DriverSetup.java:40)
at testScenarios.UrbanLadder.driverConfig(UrbanLadder.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to /192.168.1.47:5555
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
... 31 more
Caused by: java.net.ConnectException: Connection timed out: connect
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 53 more
]]>
    </error>
  </testcase> <!-- driverConfig -->
  <testcase name="driverConfig" time="21.779" classname="testScenarios.UrbanLadder">
    <error type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;LAPTOP-H8ACE30E&#039;, ip: &#039;192.168.1.49&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_261&#039;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-H8ACE30E', ip: '192.168.1.49', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
at userDefinedLibraries.DriverSetup.driverInstantiate(DriverSetup.java:56)
at testScenarios.UrbanLadder.driverConfig(UrbanLadder.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to /192.168.1.47:5555
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
... 31 more
Caused by: java.net.ConnectException: Connection timed out: connect
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 53 more
]]>
    </error>
  </testcase> <!-- driverConfig -->
  <testcase name="driverConfig" time="21.053" classname="testScenarios.UrbanLadder">
    <error type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;LAPTOP-H8ACE30E&#039;, ip: &#039;192.168.1.49&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_261&#039;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-H8ACE30E', ip: '192.168.1.49', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
at userDefinedLibraries.DriverSetup.driverInstantiate(DriverSetup.java:40)
at testScenarios.UrbanLadder.driverConfig(UrbanLadder.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to /192.168.1.47:5555
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
... 31 more
Caused by: java.net.ConnectException: Connection timed out: connect
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 53 more
]]>
    </error>
  </testcase> <!-- driverConfig -->
  <testcase name="operationSearchBar" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSearchBar -->
  <testcase name="operationSearchBar" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSearchBar -->
  <testcase name="operationSearchBar" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSearchBar -->
  <testcase name="operationSearchBar" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSearchBar -->
  <testcase name="operationCategoryMenu" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationCategoryMenu -->
  <testcase name="operationCategoryMenu" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationCategoryMenu -->
  <testcase name="operationCategoryMenu" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationCategoryMenu -->
  <testcase name="operationCategoryMenu" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationCategoryMenu -->
  <testcase name="operationPriceSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationPriceSelect -->
  <testcase name="operationPriceSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationPriceSelect -->
  <testcase name="operationPriceSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationPriceSelect -->
  <testcase name="operationPriceSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationPriceSelect -->
  <testcase name="operationStorageSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationStorageSelect -->
  <testcase name="operationStorageSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationStorageSelect -->
  <testcase name="operationStorageSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationStorageSelect -->
  <testcase name="operationStorageSelect" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationStorageSelect -->
  <testcase name="print" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- print -->
  <testcase name="print" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- print -->
  <testcase name="print" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- print -->
  <testcase name="print" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- print -->
  <testcase name="testCaseReadExl" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testCaseReadExl -->
  <testcase name="testCaseReadExl" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testCaseReadExl -->
  <testcase name="operationSearch" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSearch -->
  <testcase name="operationSearch" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSearch -->
  <testcase name="operationCategory" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationCategory -->
  <testcase name="operationCategory" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationCategory -->
  <testcase name="operationPrice" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationPrice -->
  <testcase name="operationPrice" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationPrice -->
  <testcase name="operationStorage" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationStorage -->
  <testcase name="operationStorage" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationStorage -->
  <testcase name="operationstock" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationstock -->
  <testcase name="operationstock" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationstock -->
  <testcase name="toExcelWrite" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- toExcelWrite -->
  <testcase name="toExcelWrite" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- toExcelWrite -->
  <testcase name="getBeingAtHomeCollections" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- getBeingAtHomeCollections -->
  <testcase name="getBeingAtHomeCollections" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- getBeingAtHomeCollections -->
  <testcase name="testcase_clickGiftCards" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_clickGiftCards -->
  <testcase name="testcase_clickGiftCards" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_clickGiftCards -->
  <testcase name="testcase_selectGiftCards" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_selectGiftCards -->
  <testcase name="testcase_selectGiftCards" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_selectGiftCards -->
  <testcase name="testcase_customiseGiftCard" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_customiseGiftCard -->
  <testcase name="testcase_customiseGiftCard" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_customiseGiftCard -->
  <testcase name="testcase_FillDetails" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_FillDetails -->
  <testcase name="testcase_FillDetails" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_FillDetails -->
  <testcase name="errorMessage" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- errorMessage -->
  <testcase name="errorMessage" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- errorMessage -->
  <testcase name="write_ErrorMessage" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- write_ErrorMessage -->
  <testcase name="write_ErrorMessage" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- write_ErrorMessage -->
  <testcase name="ScreenShot" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- ScreenShot -->
  <testcase name="ScreenShot" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- ScreenShot -->
  <testcase name="testcase_readExcelDataValid" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_readExcelDataValid -->
  <testcase name="testcase_readExcelDataValid" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_readExcelDataValid -->
  <testcase name="testcase_FillValidDetails" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_FillValidDetails -->
  <testcase name="testcase_FillValidDetails" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- testcase_FillValidDetails -->
  <testcase name="proceedToPayment" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- proceedToPayment -->
  <testcase name="proceedToPayment" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- proceedToPayment -->
  <testcase name="studyChairSearch" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- studyChairSearch -->
  <testcase name="studyChairSearch" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- studyChairSearch -->
  <testcase name="studyChairSearch" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- studyChairSearch -->
  <testcase name="studyChairSearch" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- studyChairSearch -->
  <testcase name="operationSortBy" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSortBy -->
  <testcase name="operationSortBy" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSortBy -->
  <testcase name="operationSortBy" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- operationSortBy -->
  <testcase name="StudyChairprint" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- StudyChairprint -->
  <testcase name="StudyChairprint" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- StudyChairprint -->
  <testcase name="StudyChairprint" time="0.000" classname="testScenarios.UrbanLadder">
    <skipped/>
  </testcase> <!-- StudyChairprint -->
</testsuite> <!-- testScenarios.UrbanLadder -->
